third_party/pcsc-lite/naclport/js_client/src/api.js

```diff
@@ -2205,6 +2206,29 @@ goog.exportProperty(
     'getResult',
     API.ResultOrErrorCode.prototype.getResult);
 
+/**
+ * Generates a debug textual representation of the PC/SC-Lite call result
+ * structure.
+ *
+ * This function is safe to be used in Release builds, because all potentially
+ * privacy-sensitive data is stripped away from the resulting text.
```

Comment:
I guess dump() just returns the content, but |errorCode| doesn't contain any privacy-sensitive information and debugDump() is responsible for removing the potential privacy-sensitive information?
optional: I would explicitely mention the use of debugDump() in this comment then

Reply:
Done - added inline comments into the implementation.